roboteam_ai/src/world/Robot.cpp

```diff
@@ -253,6 +278,20 @@ void Robot::setBatteryLow(bool batteryLow) {
     Robot::batteryLow = batteryLow;
 }
 
+/*
+ * Returns true when a feedback packet has been detected at most 2 second ago.
+ */
+bool Robot::hasRecentFeedback() {
+    return (lastReceivedFeedbackMoment > world->getTime() - 1.0);
```

Comment:
Preferably internal system clock time is used, since this is safer on the network. This should work, but it's safer.